.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: c
```

Comment:
Unfortunately we can't build and run tests on travis because at the moment they only support Mono and .NET Core, hence the basic setup for c to simply run the dev checks.